gazebo_msgs/CMakeLists.txt

```diff
@@ -2,6 +2,20 @@ cmake_minimum_required(VERSION 3.5)
 
 project(gazebo_msgs)
 
+if(NOT DEFINED IGNORE_GAZEBO_MSGS_WARNING)
```

Comment:
Most users will not be building this package from source so they will not be seeing this warning. Ideally, we would print the warning when the downstream application is doing a `find_package`.

Reply:
Chris suggested using `CONFIG_EXTRAS` with `ament_package` , e.g https://github.com/ros2/rclcpp/blob/rolling/rclcpp_components/CMakeLists.txt#L176

Reply:
good catch. [7037648](https://github.com/ros-simulation/gazebo_ros_pkgs/pull/1533/commits/70376483630d04361461f4e00e13a1920ac48a99) Still need to test it, closing by today.

Reply:
Okay, thanks. I'd be okay with removing this and only doing the `CONFIG_EXTRAS` thing, but I'll leave it up to you.